ec/src/models/short_weierstrass/affine.rs

```diff
@@ -305,14 +305,14 @@ impl<P: SWCurveConfig, T: Borrow<Self>> Sub<T> for Affine<P> {
 impl<P: SWCurveConfig> Sub<Projective<P>> for Affine<P> {
     type Output = Projective<P>;
     fn sub(self, other: Projective<P>) -> Projective<P> {
-        other - self
+        self.into_group() - other
```

Comment:
```suggestion
        self + (-other)
```

Reply:
Thanks for pointing out the better solution! Wasn't aware that we can do this 😂